File: Modul-5/Soal-5/PRAK505-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>
void Biodata(int tahunlahir, char nama[], char asal[]){
    int tahun_sekarang = 2020;
    printf("-----------------------------------\n");
    printf("Perkenalkan Nama Saya: %s\n", nama);
    printf("Umur Saya : %d\n", tahun_sekarang-tahunlahir);
    printf("Saya Adalah Angkatan: %d\n", tahun_sekarang);
    printf("Asal Saya dari: %s\n", asal);
}
int main() {
    int tahunLahir;
    char Namaku[20], Asal[15];
    scanf(" %d",&tahunLahir);
    scanf(" %[^\n]%*c",&Namaku);
    scanf(" %[^\n]%*c",&Asal);
    Biodata(tahunLahir, Namaku, Asal);
return 0;
}

File: Modul-5/Soal-5/PRAK505-2410817110016-MuhammadAzriRahman.py
def Biodata(tahunlahir, nama, asal):
    tahun_sekarang = 2020
    print("-----------------------------------\n");
    print(f"Perkenalkan Nama Saya: {nama}");
    print(f"Umur Saya : {tahun_sekarang-tahunlahir}");
    print(f"Saya Adalah Angkatan: {tahun_sekarang}");
    print(f"Asal Saya dari: {asal}");

if __name__ == "__main__":
    tahunLahir = int(input(""))
    Namaku = input("")
    Asal = input("")

    Biodata(tahunLahir, Namaku, Asal);
